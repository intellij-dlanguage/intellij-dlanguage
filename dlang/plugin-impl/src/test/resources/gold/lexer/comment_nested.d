/+
 hello
 +/

// transition line comment
/++/

/+ Fist level
/+ Second level +/
// Still in block
+/
// transition line comment
