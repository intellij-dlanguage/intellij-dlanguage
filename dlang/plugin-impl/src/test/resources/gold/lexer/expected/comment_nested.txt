DlangTokenType.NESTING_BLOCK_COMMENT ('/+\n hello\n +/')
WHITE_SPACE ('\n\n')
DlangTokenType.LINE_COMMENT ('// transition line comment')
WHITE_SPACE ('\n')
DlangTokenType.NESTING_BLOCK_COMMENT ('/++/')
WHITE_SPACE ('\n\n')
DlangTokenType.NESTING_BLOCK_COMMENT ('/+ Fist level\n/+ Second level +/\n// Still in block\n+/')
WHITE_SPACE ('\n')
DlangTokenType.LINE_COMMENT ('// transition line comment')